nodejs/trivialOdbc/.odbcini

```diff
@@ -0,0 +1 @@
+DRIVER=IBM i Access ODBC Driver
```

Comment:
This file is not required. The .odbc.ini file is placed in the user's home directory.